src/main/resources/standardTestCases/non_root_default_user/non_root_default_user.wdl

```diff
@@ -0,0 +1,21 @@
+task notroot {
+  command {
+    echo $HOME
+  }
+
+  runtime {
+    docker: "mcovarr/notroot:v1"
```

Comment:
I missed the point about the Dockerfile earlier, where do you think is an appropriate place for that?